Gemfile

```diff
@@ -25,9 +25,7 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 
 # For styling
 gem 'bootstrap-sass', '~> 3.3', '>= 3.3.5.1'
-
-
-
+gem 'carrierwave', '~> 0.9'
```

Comment:
I was thinking about allowing people to upload media for movies, books, and albums, but gave up on it after starting the project.